recon-enumeration/index.md

```diff
@@ -0,0 +1,211 @@
+DIRB comes with a set of preconfigured attack wordlists for easy usage but you can use your custom wordlists.
+For each filename, it checks the existence on the web server and returns the results which do not give a 404 Not Found response.
+
+Usage example: `./dirb <url_base> [<wordlist_file(s)>] [options]`
+
+![DIRB example](./support/dirb_example.png)
+
+The output lines with the results found (not 404) start with a `+` and give details about status code and page size.
+
+You can read the documentation if you want to specify custom options, like custom file extensions to look for.
+
+### Similar tools
+* [DirBuster](https://tools.kali.org/web-applications/dirbuster) - not maintained anymore, Kali built in, written in Java.
+The only one with a GUI and not a CLI.
+* [DirSearch](https://www.kali.org/tools/dirsearch/) - Kali built in & on [GitHub]((https://github.com/maurosoria/dirsearch)), written in Python.
+* [GoBuster](https://tools.kali.org/web-applications/gobuster) - Kali built in & on [GitHub](https://github.com/OJ/gobuster), written in Go.
+Can do DNS and VHost busting, S3 buckets enumeration.
+* [wfuzz](https://github.com/xmendez/wfuzz) - on GitHub, written in Python, can be easily installed with pip.
+* [ffuf](https://github.com/ffuf/ffuf) - on GitHub, written in Go, has the option to mutate the files found.
+Can do VHost discovery.
+
+You can find a detailed comparison [here](https://blog.sec-it.fr/en/2021/02/16/fuzz-dir-tools/).
+We don't recommend DirBuster and DirSearch since the others are newer, improved and better-maintained tools.
+DIRB is also not maintained anymore, but can come in handy if you want to do a quick content discovery, with not many options to configure.
+
+## Recap with DVWA
+
+DVWA (Damn Vulnerable Web Application) is a deliberately vulnerable application which you can use to test your skills and exploit some of the most common web vulnerabilities, like XSS, SQL, LFI, with various difficulty levels.
+
+### Setup instructions
+
+An easy way to deploy DVWA is by using [Docker](https://docs.docker.com/get-docker/), which you need to have installed on your machine.
+Then, follow the instructions from [here](https://hub.docker.com/r/vulnerables/web-dvwa/).
+After running it, you can simply access it in your browser at http://127.0.0.1:80 (or a different port, if you wish), login and start hacking.
+
+### Solutions
+* Brute force: https://medium.com/cyberdefendersprogram/burp-suite-webpage-enumeration-and-vulnerability-testing-cfd0b140570d
+* SQL Injection: https://medium.com/cyberdefendersprogram/burp-suite-webpage-enumeration-and-vulnerability-testing-cfd0b140570d
+* Blind Sql: https://medium.com/hacker-toolbelt/dvwa-1-9-viii-blind-sql-injection-with-sqlmap-ee8d59fbdea7
+* CSRF: https://medium.com/hacker-toolbelt/dvwa-1-9-iv-csrf-b9f2409fd628
+* File Upload: https://medium.com/hacker-toolbelt/dvwa-1-9-vi-file-upload-ce77a73b0453
```

Comment:
I don't know if it's better to show them the solutions or let them search for it, but we can leave it like this for the moment.